test/storageDataGroup.js

```diff
@@ -45,6 +45,8 @@ describe('StorageDataGroup', () => {
             '        }',
             '    }',
             '    partition appsvcs',
+            '    description this is my group',
+            '    app-service my app service',
```

Comment:
I agree I think just changing this specific piece is good, instead of trying to make something more generic.